scripts/17--html--forms/README.md

```diff
@@ -92,50 +118,56 @@
     </select>
     ```
 
-    - also `<optgroup>` for grouping options
-    - more advanced fake select boxes using JS
+- also `<optgroup>` for grouping options
+- more advanced fake select boxes using JS
 
-- Text Area
+#### Textarea tag
 
-    ```html
-    <label for="message">Textarea</label>
-    <textarea id="message" name="message"></textarea>
-    ```
+- `<textarea>`
+- not self-closing
+- can enter multi-line text for eg. a comment
+- uses all the same attributes as `<input>`
+- however `value` goes *between* tags
```

Comment:
I'd maybe stick with `"value"` – it's not actually a `value` attribute, just does the equivalent of one.